src/routes/help/components.svelte

```diff
@@ -0,0 +1,243 @@
+<script>
+	import components from '../components/components.json';
+
+	const allTags = Array.from(new Set(components.map((item) => item.tags).flat()));
+	const tagItems = allTags.map((t) => ({ label: t, value: t }));
+	const allCategories = [
+		'Boilerplate',
```

Comment:
I don't see any category called "Boilerplate". Where is this coming from?

Reply:
It is in the current staging site. This is the reason I didn't make the categories dynamic. There is this inconsistency here, on `/help/components` the list is `Boilerplate, Data Visualisation, Design Pattern, Design System, Developer Experience, Forms & User Input, Integration, Routers, Stores, Testing, User Interaction` but on `/components` the dynamically generated list contains `Forms & User Input, User Interaction, SvelteKit Adapters, Routers, Integration, Developer Experience, Design System, Design Pattern, Stores, Data Visualisation, Testing, Animations, Rich Text Editor`. I'm not sure which one to follow.

![image](https://user-images.githubusercontent.com/34279631/132468646-06bd97d7-30ae-4e15-be83-98de4837f5aa.png)

Reply:
`/help/components` is outdated. Please use the list from `/components` instead. That should allow you to make it dynamic as well

Reply:
Committed. 
Should I sort categories and tags alphabetically or leave them as is? I assume it'll be easier to scan if it's sorted.

Reply:
sorting makes sense to me

Reply:
Committed.